### DISCUSSION TITLE
allow passing options to `next lint`

### DISCUSSION THREAD

**Discussion by stefanprobst** [vercel/next.js#25937] [Ideas]
### Describe the feature you'd like to request

currently i'm running eslint with `--cache` and `--ignore-path` options, but `next lint` currently only accepts a `--dir` option, and fails when trying to pass additional options to eslint.

### Describe the solution you'd like

allow passing additional options to eslint with `next lint`.

### Describe alternatives you've considered

use `eslint` directly. (are there advantages to `next lint` over directly using the eslint cli, other than initial setup guidance?)

**Comment by housseindjirdeh** [comment]
Done here: #25952. Most of the options can now be passed directly :)
